Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/TypeContentsOrderConfiguration.swift

```diff
@@ -40,6 +41,7 @@ struct TypeContentsOrderConfiguration: SeverityBasedRuleConfiguration {
         [.ibAction],
         [.otherMethod],
         [.subscript],
-        [.deinitializer]
+        [.deinitializer],
+        [.ibSegueAction]
```

Comment:
This needs to be removed. We try to avoid changing the defaults subsequently if possible.